plugins/ipam/static/README.md

```diff
@@ -60,3 +60,9 @@ The plugin also support following [capability argument](https://github.com/conta
 The following [args conventions](https://github.com/containernetworking/cni/blob/master/CONVENTIONS.md#args-in-network-config) are supported:
 
 * `ips` (array of strings): A list of custom IPs to attempt to allocate, with prefix (e.g. '10.10.0.1/24')
+
+Note: If some of above is used at same time, one input sources are used as following priority:
+
+1. [args conventions](https://github.com/containernetworking/cni/blob/master/CONVENTIONS.md#args-in-network-config)
+1. [capability argument](https://github.com/containernetworking/cni/blob/master/CONVENTIONS.md)
```

Comment:
@s1061123 Casey and I think capability args should be #1, then args, then CNI_ARGS. Does that look OK?

Reply:
Sure. Changed.